src/core/stream/segment_bookkeeper.ts

```diff
@@ -659,6 +659,17 @@ export default class SegmentBookkeeper {
     }
   }
 
+  public getBitrate(currentTime: number){
```

Comment:
spacing

Reply:
also the function is too ad-hoc and the name is too generic.

Maybe ``getSegmentInfosForTime(time : number) : gnagna|null``